src/themes/ec/variables/_featured-item.scss

```diff
@@ -9,8 +9,7 @@ $featured-item: (
   max-width: 918px,
   title: (
     color: var(--cm-on-surface-brand),
-    font-mobile: var(--f-2xl),
-    font-desktop: var(--f-3xl),
+    font: '3xl',
```

Comment:
do you think it would be possible to still use `var(--f-3xl)`, and have the mixin handle it correctly?
that way we would not have to figure out if we should put a var or a direct value